services/tables/src/main/resources/application.properties

```diff
@@ -1,5 +1,6 @@
 spring.mvc.throw-exception-if-no-handler-found=true
-spring.web.resources.add-mappings=false
+spring.mvc.static-path-pattern=favicon.ico
```

Comment:
this property allows the favicon to be presented to browsers and thus fixes issue with metrics/logs spamming errors when trying to load the favicon via swagger or /actuator/prometheus

context for the old property is:

> We have seen a few instances where the malformed/wrong URL causes 404 error code and makes users believe they are actually requesting some resource that doesn't exist any more. However this is obviously different from true positive case where request resource doesn't exist. This problem is more severe with SQL API on top of REST API, as SQL API will simply translate 404 to "Table Not found" whereas it should be "URL malformed / Bad Request"
This PR addresses that by customizing handling of "NoHandlerFoundException" which by default is treated as 404 by springboot framework.

which this method _should_ solve both of above issues.